spec/integration/federation/receive_federation_messages.rb

```diff
@@ -0,0 +1,93 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.